source/mapAccumRight.js

```diff
@@ -26,27 +26,27 @@ import _curry3 from './internal/_curry3';
  * @example
  *
  *      var digits = ['1', '2', '3', '4'];
- *      var append = (a, b) => [a + b, a + b];
+ *      var append = (a, b) => [b + a, b + a];
```

Comment:
this won't work in the repl because `append` is already defined. please give `append` a different name. i realize you didn't introduce this bug, but as long as you are in there ...

Reply:
Ah yes I noticed that bug but forgot to fix, thank you for reminding me.